OpenSim/Simulation/Model/ContactGeometry.h

```diff
@@ -46,7 +46,7 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(ContactGeometry, ModelComponent);
 // DATA
 //=============================================================================
 protected:
-    Body* _body;
+    SimTK::ReferencePtr<PhysicalFrame> _body;
```

Comment:
@chrisdembia I cannot use `T = const PhysicalFrame` in this case because ContactGeometry directly updates the displayer (VisibleObject) that is on the Body (which I think is bad) but the VisibleObject stuff is supposed to go away with Ayman's new Geometry.